t/test_validate_syntax.t

```diff
@@ -23,7 +23,7 @@ engine = SQLite
 database_file = $tempdir/zonemaster.sqlite
 
 [LANGUAGE]
-locale = en_US fr_FR da_DK fi_FI nb_NO sv_SE
+locale = da_DK en_US es_ES fi_FI fr_FR nb_NO sv_SE
```

Comment:
`fr_FR` and `nb_NO` should be enough here.

Reply:
If it is not necessary to have all locale here it is out of scope to update it. I will just remove it from the PR with a force push. It could be updated in another PR concerning the unit test.